wdl/meta.wdl

```diff
@@ -256,20 +276,35 @@ task meta_qq {
 
         mv ~{meta_file} ~{base}
 
-        /META_ANALYSIS/scripts/qqplot.R --file ~{base} --bp_col "POS" --chrcol "#CHR" --pval_col ~{pvals_to_plot} --loglog_ylim ~{loglog_ylim}
+        [[ "~{pvals_to_plot}" =~ "leave_" ]] && loo="--loo" || loo=""
```

Comment:
you could print with the error message here and explicit return error code in case wrong combo of switches are used. This is pretty cryptic

Reply:
My idea here was that we wouldn't need to manually set an extra switch to say to use the loo results as well but just check that if we want to plot them, then we want to qc them as well.

But true this starts to bee a little too cryptic..